keystrokes/src/components/Character/CharacterCreationModal.js

```diff
@@ -0,0 +1,118 @@
+    }
+
+    return (
+        <Modal
+            open={isOpen}
+            onClose={setIsOpen(true)}
+            className={classes.modal}
+        >
+            <div className={classes.paper}>
+            <Grid container spacing={2} alignItems="center">
+                <Grid item>
+                    <TextField
+                        label="Character Name"
+                        value={characterName}
+                        onChange={handleChangeName}
+                        variant="outlined"
+                        className={classes.button}
+                    />
+                </Grid>
+                <Grid item>
+                <FormControl className={classes.button}>
+                    <InputLabel>Race</InputLabel>
+                    <Select
+                        value={characterRace}
+                        onChange={handleChangeRace}
+                    >
+                        {
+                            characterDetails.race.map(race => (
+                                <MenuItem key={race} value={race}>
+                                    {race}
+                                </MenuItem>
+                            ))
+                        }
+                    </Select>
+                </FormControl>
+                </Grid>
+                <Grid item>
+                <FormControl className={classes.button}>
+                    <InputLabel>Class</InputLabel>
+                    <Select
+                        value={characterClass}
+                        onChange={handleChangeClass}
+                    >
+                        {
+                            Object.keys(characterDetails.class).map(heroClass => (
+                                <MenuItem key={heroClass} value={heroClass}>
+                                    {heroClass}
+                                </MenuItem>
+                            ))
+                        }
+                    </Select>
+                </FormControl>
+                </Grid>
+                <Grid item>
+                <FormControl disabled={characterClass === ""} className={classes.button}>
+                    <InputLabel>Spec</InputLabel>
+                    <Select
+                        value={characterSpec}
+                        onChange={handleChangeSpec}
+                    >
+                        {
+                            characterDetails.class[characterClass].map(spec => (
+                                <MenuItem key={spec} value={spec}>
+                                    {spec}
+                                </MenuItem>
+                            ))
+                        }
+                    </Select>
+                </FormControl>
+                </Grid>
+                </Grid>
+                <Grid container justify="flex-end">
+                <Grid item style={{paddingTop: '5rem'}}>
+                    <Button size="large" onClick={()=> setIsOpen(false)}>Cancel</Button>
```

Comment:
`()=> setIsOpen(false)` could imply that setIsOpen returns a value. 
prefer being explicit like 
`() => { setIsOpen(false) }`